app/_hub/critical-blue/approov/index.md

```diff
@@ -3,53 +3,63 @@ name: Approov Mobile App Attestation
 publisher: CriticalBlue Ltd
 
 categories:
-  - authentication
   - security
 
 type: integration
 
-desc: Ensure that only genuine mobile app instances can connect to your server or cloud backend
+desc: Approov ensures that only genuine and unmodified instances of your mobile app can connect to your server or cloud backend.
 
 description: |
-  With Approov, you control which apps can access your mobile app backend API in a secure and easily deployable manner. Users can confidently allow API access from iOS and Android devices knowing that Approov will only authenticate legitimate apps and does not rely on app-embedded secrets or keys.
+With Approov you control what can access your mobile app backend API in a secure and easily deployable manner. Our customers confidently allow API access from iOS and Android devices knowing that Approov will only authenticate legitimate instances of your mobile apps without relying on embedded secrets or keys stored in the app itself.
 
-  This capability prevents the misuse of your API by either automated software agents or unauthorized third-party apps, providing the basis for a range of API access management policies.
+This capability prevents misuse of your API by either automated software agents or unauthorized third-party apps, providing the basis for a range of API access management policies.
 
 support_url: https://approov.zendesk.com/hc/en-gb/requests/new
 
 source_url: https://github.com/approov/kong_approov-plugin
 
+
 kong_version_compatibility:
   community_edition:
     compatible:
+      - 0.15.x
       - 0.14.x
       - 0.13.x
       - 0.12.x
       - 0.11.x
       - 0.10.x
   enterprise_edition:
     compatible:
-      - 0.34-x
-      - 0.33-x
-      - 0.32-x
+      - 1.5.x
+      - 1.3.x
```

Comment:
the enterprise versions do not match the CE versions? if EE is only those, than CE should be 1.3+